refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsMethodInvocationWithTwoOrMoreArgsTest.java

```diff
@@ -0,0 +1,73 @@
+package tech.picnic.errorprone.refaster.matchers;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.bugpatterns.BugChecker;
+import org.junit.jupiter.api.Test;
+
+final class IsMethodInvocationWithTwoOrMoreArgsTest {
+  @Test
+  void matches() {
+    CompilationTestHelper.newInstance(
+            IsMethodInvocationWithTwoOrMoreArgsTest.MatcherTestChecker.class, getClass())
```

Comment:
```suggestion
            MatcherTestChecker.class, getClass())
```
We can drop the qualifier.